DiscordBot/Commands/EventCommands.cs

```diff
@@ -90,6 +99,117 @@ public async Task CreateEvent(CommandContext context, InteractivityModule intera
             await context.RespondAsync(embed: embed);
         }
 
+        public async Task EditEvent(CommandContext context, InteractivityModule interactivity)
+        {
+            await context.RespondAsync($"{context.Member.Mention} - what is the event key?");
+            var eventKey = await GetUserIntResponse(context, interactivity);
+            if (eventKey == null)
+            {
+                return;
+            }
+
+            var embed = await GetEventEmbed(context, eventKey.Value);
+            if (embed == null)
+            {
+                return;
+            }
+
+            await EditEventFields(context, interactivity, eventKey.Value, embed);
+        }
+
+        private static async Task EditEventFields(
+            CommandContext context,
+            InteractivityModule interactivity,
+            int eventKey,
+            DiscordEmbedBuilder embed)
+        {
+            string? newName = null;
+            string? newDescription = null;
+            DateTimeOffset? newTime = null;
```

Comment:
These variables might not be needed once GetEvent is implemented since values from that can be used instead.

GetEvent currently returns hardcoded values which will overwrite the existing entry.